examples/src/main/java/io/dapr/examples/invoke/grpc/README.md

```diff
@@ -95,55 +93,59 @@ The `app-id` argument is used to identify this service in Dapr's runtime. The `a
 
 ### Running the example's client
 
-The other component is the client. It will send one message per second to the service via Dapr's invoke API using Dapr's SDK. Open the `HelloWorldClient.java` file, it uses the Dapr's Java SDK to invoke the `say` method on the service above:
+The other component is the client. It will add user name to the grpc request and send it to the server. Open the `HelloWorldClient.java` file, it creates a new grpc channel and sends request directly to the dapr side car through this channel.
 
 ```java
 private static class HelloWorldClient {
 ///...
   public static void main(String[] args) throws Exception {
-    try (DaprClient client = new DaprClientBuilder().build()) {
-    
-      String serviceAppId = "hellogrpc";
-      String method = "say";
-
-      int count = 0;
-      while (true) {
-        String message = "Message #" + (count++);
-        System.out.println("Sending message: " + message);
-        client.invokeMethod(serviceAppId, method, message, HttpExtension.NONE).block();
-        System.out.println("Message sent: " + message);
-
-        Thread.sleep(1000);
-
-        // This is an example, so for simplicity we are just exiting here.  
-        // Normally a dapr app would be a web service and not exit main.
-        System.out.println("Done");
+
+    String user = "World";
+    // Access a service running on the local machine on port 50051
+    String target = "localhost:50051";

```

Comment:
Consume from DAPR_GRPC_PORT env that is injected automatically and fallback to the default port.

Reply:
Modified, Thanks.